spec/services/hyrax/file_set_derivatives_service_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'rails_helper'
 
-RSpec.describe Hyrax::FileSetDerivativesService do
+RSpec.describe Hyrax::DerivativeService do
```

Comment:
@dlpierce Should this file then get renamed `DerivativeService`?

Reply:
Yeah, I'll do that.